compiler/rustc_parse_format/src/lib.rs

```diff
@@ -750,6 +761,29 @@ impl<'a> Parser<'a> {
         }
         if found { Some(cur) } else { None }
     }
+
+    fn suggest_positional_arg_instead_of_captured_arg(&mut self, arg: Argument<'a>) {
+        if let Some(end) = self.consume_pos('.') {
+            let byte_pos = self.to_span_index(end);
+            let start = InnerOffset(byte_pos.0 + 1);
+            let field = self.argument(start);
+            if let ArgumentNamed(_) = arg.position {
+                if let ArgumentNamed(_) = field.position {
```

Comment:
```suggestion
            if let ArgumentNamed(_) = arg.position
                && let ArgumentNamed(_) = field.position
            {
```

Reply:
Please use let chains to reduce indentation.

Reply:
`rustc_parse_format` does not have let_chains feature and It seems I should not add this feature due to the following comment.

https://github.com/rust-lang/rust/blob/2f2243c9b6686d2314515561a80898e746015231/compiler/rustc_parse_format/src/lib.rs#L12-L13

Reply:
ok, then it's fine :)